src/shammodels/gsph/src/modules/GSPHSetup.cpp

```
@@ -0,0 +1,184 @@
+        if (shamcomm::world_rank() == 0) {
+            logger::normal_ln("SPH setup", "dumping setup log to ", fname);
+        }
+
+        std::ofstream file(fname);
+        file << json_data.dump(4);
+        file.close();
+
+        step_counter++;
+    }
+
+    void update_count_per_rank(u64 count) {
```

Cppcheck (2.21.0)

[style] The function 'update_count_per_rank' is never used.

(unusedFunction)